docker-compose.yml

```diff
@@ -319,6 +323,16 @@ services:
       - "/config/admin_user_cred.json"
     << : *log-config
 
+  state_groups_cleaner:
```

Comment:
IMO it would make sense to also have `restart: always` here.

Reply:
Not entirely sure: `restart: always` would force us to come up with a more involved timing scheme, otherwise we risk that the initial short (<=24h) delay is always used and we'd kill the `synapse` stack at least daily.

Reply:
Okay, I've added an "early restart" check and have the `restart: always` clause enabled. [The commit](https://github.com/konradkonrad/raiden-service-bundle/commit/dd291fbb04e81ed1bc7100924b389e3b08f7fd05) doesn't show up in my browser in this PR, but it is on the branch in my fork.